data/theme/gnome-shell.css

```diff
@@ -690,8 +690,8 @@ StScrollBar {
       -panel-corner-background-color: transparent;
       -panel-corner-border-color: transparent; }
   #panel .panel-button {
-    -natural-hpadding: 12px;
-    -minimum-hpadding: 6px;
+    -natural-hpadding: 10px;
+    -minimum-hpadding: 10px;
```

Comment:
Please move these changes to an Endless specific section (after the `/* Endless-specific overrides below this point *` comment below), to help with future rebases.

Reply:
Thanks, I didn't actually notice we had an Endless-specific section!